documentation/book/proc-metrics-deploying-prometheus.adoc

```diff
@@ -17,48 +17,44 @@ NOTE: By default, the Prometheus Operator only supports jobs that include an `en
 
 .Procedure
 
-. Download the Prometheus resource file from the repository and replace the example namespace with your own:
+. Modify the Prometheus installation file (`prometheus.yaml`) according to the namespace Prometheus is going to be installed in:
 +
 On Linux, use:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+sed -i 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
 +
 On MacOS, use:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+sed -i '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
 
-. Download and edit the `ServiceMonitor` resource in `strimzi-service-monitor.yaml` to define Prometheus jobs that will scrape the metrics data:
-+
-[source,shell,subs="+quotes,attributes"]
-curl -O https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/strimzi-service-monitor.yaml
+. Edit the `ServiceMonitor` resource in `strimzi-service-monitor.yaml` to define Prometheus jobs that will scrape the metrics data.
 
 . To use another role:
-+
+
 .. Create a `Secret` resource:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -O https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/additional-properties/prometheus-additional.yaml
 oc create secret generic additional-scrape-configs --from-file=prometheus-additional.yaml
-+
+
 .. Edit the `additionalScrapeConfigs` property in the `prometheus.yaml` file to include the name of the `Secret` and the YAML file (`prometheus-additional.yaml`) that contains the additional configuration.
 
 . Download and edit the `prometheus-rules.yaml` file that creates sample alert notification rules:
 +
 On Linux, use:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
+sed -i 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
 +
 On MacOS, use:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
+sed -i '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
```

Comment:
I am not sure about mac version, but the same comment about `>` might apply there.